core/block/import/markdown/blockconverter.go

```diff
@@ -61,14 +57,14 @@ func (m *mdConverter) processFiles(importPath string, mode string, allErrors *ce
 	supportedExtensions = append(supportedExtensions, videoFormats...)
 	supportedExtensions = append(supportedExtensions, imageFormats...)
 	supportedExtensions = append(supportedExtensions, audioFormats...)
-	readers, err := s.GetFileReaders(importPath, supportedExtensions, nil)
+	readers, err := importSource.GetFileReaders(importPath, supportedExtensions, nil)
 	if err != nil {
 		allErrors.Add(err)
 		if mode == pb.RpcObjectImportRequest_ALL_OR_NOTHING.String() {
 			return nil
 		}
 	}
-	if len(readers) == 0 {
+	if source.CountFilesWithGivenExtension(readers, ".md") == 0 {
```

Comment:
We consider, that there is no files to import in case, when directory or zip archive doesn't have any `md` files. If directory or zip archive contain other files, like images or video, it still means, that there is no file to import.